cron/daily/100-pending-orders-reminder.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+import '../../server/env';
+
+import models, { Op } from '../../server/models';
+import status from '../../server/constants/order_status';
+import * as libPayments from '../../server/lib/payments';
+
+const REMINDER_DAYS = 4;
+
+const fetchPendingOrders = async date => {
+  const dateFrom = new Date(date).setUTCHours(0, 0, 0, 0);
+  const dateTo = new Date(dateFrom).setUTCHours(23, 59, 59);
+
+  const orders = await models.Order.findAll({
+    where: {
+      status: status.PENDING,
+      deletedAt: null,
+      PaymentMethodId: null,
+      createdAt: { [Op.gte]: dateFrom, [Op.lte]: dateTo },
+    },
+    include: [
+      { model: models.Collective, as: 'fromCollective' },
+      { model: models.User, as: 'createdByUser' },
+      { model: models.Collective, as: 'collective' },
+    ],
+  });
+
+  return orders;
+};
+
+const run = async () => {
+  const reminderDate = process.env.START_DATE ? new Date(process.env.START_DATE) : new Date();
+  reminderDate.setDate(reminderDate.getDate() - REMINDER_DAYS);
+
+  const orders = await fetchPendingOrders(reminderDate);
+  orders.forEach(async order => {
+    await libPayments.sendReminderPendingOrderEmail(order);
+  });
+};
+
+run();
```

Comment:
Please rename it as `80` instead of `100`. We're relying on that to run the jobs sequentially. A bit old school but ...

Reply:
Changed.